pkg/telemetry/manager.go

```diff
@@ -198,10 +198,11 @@ func (m *manager) workflowsLoop() {
 		}
 	}()
 
+forLoop:
 	for {
 		select {
 		case <-m.done:
-			break
+			break forLoop
```

Comment:
Nice catch!
Since nothing happens after this for loop, maybe just `return`?

What about line `231`? It breaks out the case statement. If it's desired I'd remove explicit `break`. WDYT?

Reply:
yup works as expected, but it's just redundant. Without it it works exactly the same

Reply:
Let's leave it as is then. If we ever go around to change it then we change revise it.